CHANGELOG.md

```diff
@@ -4,9 +4,7 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
-## 11.0.15 - *2023-07-10*
-
-## 11.0.14 - *2023-05-17*
+- feat: add support for ubuntu 22.04
```

Comment:
Please move this under ``Unreleased`` and don't remove the other lines you removed.

Reply:
@ramereth The changelog file is correct, can you merge this PR please?